subscriptions/sales_flow/closing.rst

```diff
@@ -0,0 +1,107 @@
+====================
+Close a subscription
+====================
+
+Losing a customer is always difficult, especially if you put a lot of effort into getting them to
+sign up for your products/services. This is understandable. However, many companies come up with
+dubious methods to reduce the probability of this happening.
+
+.. raw:: html
+
+   <div align="center" style="color:#AD5E99; font-size: 2rem ;margin: 20px 0"> <b>What is the right
+   way to go?</b> </div>
+
+1. If you adopt such methods, you will be among companies that actively and indirectly spread the
+   phobia of subscriptions, who do not think about how negatively dissatisfied customers could impact
+   their business and who frustrate the customers in a way or another. However, at some point, it is
+   understandable that you do not want your customers to be involved in your subscription status.
+
+2. If you do not adopt such methods, you will be among companies that continue to evolve in a
+   long-term subscription business model, especially in an era of ultra-fast communications between
+   dissatisfied customers, and who retain their customers by making it easier for them to leave if
+   they want to. By giving them the opportunity to close their own subscriptions, your customers are
+   not feel trapped because they subscribed to your products/services.
```

Comment:
```suggestion
2. If you do not adopt such methods, you are among companies that continue to evolve in a
   long-term subscription business model, especially in an era of ultra-fast communications between
   dissatisfied customers, and who retain their customers by making it easier for them to leave if
   they want to. By giving them the opportunity to close their own subscriptions, your customers do not
    feel trapped because they subscribed to your products/services.
```